### PR TITLE
Issue/4685 force delete

### LINKED ISSUE #4685: Allow forceful delete in expert mode
In expert mode it is possible to forcefully delete an instance. 

https://github.com/inmanta/inmanta-core/issues/5123

This is not restorable so it might be good to call this "Destroy instance" with a warning and confirmation that this cannot be undone.

Image

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4685 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/Slices/ServiceInventory/UI/Components/DestroyModal/DestroyModal.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const onSubmit = async () => {
+    setIsOpen(false);
+    const result = await trigger(refetch);
+    if (Maybe.isSome(result)) {
+      setErrorMessage(result.value);
+    }
+  };
+  return (
+    <>
+      <ToastAlert
```
[the remark this answers is not in the document]
Comment (matborowczyk) [reply src/Slices/ServiceInventory/UI/Components/DestroyModal/DestroyModal.tsx:43]: I didn't noticed that I misnamed component, it has modal but on it's own it's par of expert mode Section, button, toast and modal

File: changelogs/unreleased/4682-expert-mode-bar.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+change-type: patch
+description: 'Add support for lsm expert mode'
```
[the remark this answers is not in the document]
Comment (matborowczyk) [reply changelogs/unreleased/4682-expert-mode-bar.yml]: Waiting for some answers

File: src/Slices/ServiceInventory/UI/Components/ForceStateAction.tsx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+  confirmationText: string;
+  id: string;
+  targetState: string;
+  onSetInstanceState: (targetState: string) => Promise<void>;
+  isModalOpen: boolean;
+  setIsModalOpen: (isOpen: boolean) => void;
+  setErrorMessage: (message: string) => void;
+}
+
+const ConfirmationModal: React.FC<ModalProps> = ({
```
[the remark this answers is not in the document]
Comment (matborowczyk) [reply src/Slices/ServiceInventory/UI/Components/ForceStateAction.tsx]: I moved that component to separate file

**Review by LauraStordeur — APPROVED** [review]

**Comment by matborowczyk** [comment]
https://user-images.githubusercontent.com/113331659/228485208-4b63b9b7-38b0-429f-8198-17a6d5edf38e.mov

**Comment by matborowczyk** [comment]
Screenshot 2023-03-29 at 16 00 07
last run

**Pull request closed without merging**